cfme/infrastructure/host.py

```diff
@@ -15,14 +15,20 @@
 from cfme.common.candu_views import HostInfraUtilizationView
 from cfme.common.host_views import HostAddView
 from cfme.common.host_views import HostDetailsView
+from cfme.common.host_views import HostDevicesView
```

Comment:
Might be time to just import the whole module...... optional though, and could get you into some circular imports.